Gruntfile.js

```diff
@@ -1,112 +1,115 @@
+/* globals module, require */
+
+const loadGruntTasks = require('load-grunt-tasks');
```

Comment:
Do we need this for reuse? Otherwise maybe it's a bit better for understanding the code... ;-)